language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionNodeValidator.java

```diff
@@ -291,6 +292,8 @@ public Boolean transform(FunctionSignatureNode node) {
                 && !node.closeParenToken().isMissing()
                 && node.leadingInvalidTokens().isEmpty()
                 && node.trailingInvalidTokens().isEmpty()
+                && (node.returnTypeDesc().isEmpty()
+                || node.returnTypeDesc().get().apply(this))
```

Comment:
Keeping these two in a single line is more readable IMO. WDYT?

Reply:
+1